docs/graph-homomorphism-as-natural-transformation.md

```diff
@@ -19,6 +23,12 @@ An example of this in haskell is Maybe.
  * The functor G(X) is Nothing.
  * The morphism between all F(X) and G(X) is Nothing (see [this issue though](https://github.com/matthewfranglen/category-set-graph/issues/1))
 
+[Paul: Let me translate this example into my language to see if I understand it properly.  The categories C and D are both *Set*, the functor F: *Set* -> *Set* is the identity (sends each object to itself and each morphism to itself), and the functor G: *Set* -> *Set* sends each set to the empty set and each function between sets to the empty function from the empty set to itself.  You propose to construct a natural transformation F -> G.
```

Comment:
I had thought of the source and target as different categories. In videos I have seen it's been mentioned that all types within haskell sit within a single category, and while both of these statements could be true at once I don't feel confident enough to stand on it. So in this description lets use sets.

 * The source Set, A, contains objects of any type T.
 * The target Set, B, contains a union of the `Just T` set and the `Nothing` singleton set.

Since set B is defined in terms of the set A type, and that the type for set B is thus always different, I do not believe that the functors between them can be the identity functor. You can't also define the type of set A to be a union of T and Maybe T as that makes the type of set B a union of Maybe T and Maybe Maybe T.

If you extended this Maybe wrapper count to infinity then would the two sets become equal? I think not because the type with the fewest Maybe wrappers in set A is always missing from set B. In the case of an infinite depth of Maybe wrappers would set B be a subset of set A?